### PR TITLE
Update raycaster_lite.py

### PR DESCRIPTION
**Pull request by vktcl**
## Addressed problem description

Added ray detection maximum distance limit parameter. Used to control the length of each light

### CODE REVIEW

File: nodes/analyzer/raycaster_lite.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ class SvRaycasterLiteNode(bpy.types.Node, SverchCustomTreeNode):
     bl_label = 'Raycaster'
     bl_icon = 'OUTLINER_OB_EMPTY'
     sv_icon = 'SV_RAYCASTER'
-
+    
+    MaxDistance: bpy.props.FloatProperty(name='MaxDistance', description='MaxDistance', default=1.0, update=updateNode)
```
Comment (Durman) [inline nodes/analyzer/raycaster_lite.py:34]:
```suggestion
    max_distance: bpy.props.FloatProperty(name='MaxDistance', description='MaxDistance', default=1.0, update=updateNode)
```

Comment (vktcl) [reply nodes/analyzer/raycaster_lite.py:34]: max_distance Variable already exists

Comment (Durman) [reply nodes/analyzer/raycaster_lite.py:34]: `max_distance` or `MaxDistance` is not a variable it is an attribute of the node class. Also this class has `start`, `direction` and no more other attributes. According Python convention names of a class should be in snake case. Sverchok code is not totally written according to this convention but it's better always to do for new code.

File: nodes/analyzer/raycaster_lite.py [hunk 2]
```diff
@@ -40,6 +41,7 @@ def sv_init(self, context):
 
         si('SvVerticesSocket', 'Verts')
         si('SvStringsSocket', 'Faces')
+        si('SvStringsSocket', 'MaxDistance').prop_name = 'MaxDistance' #添加输入最大距离接口
```
Comment (Durman) [inline nodes/analyzer/raycaster_lite.py:44]: This name also should be fixed

Comment (vktcl) [reply nodes/analyzer/raycaster_lite.py]: There are some errors and I need to close the merge again

**Pull request closed without merging**